tests/kernel/timer/timer_api/kernel.timer.tickless fails to build on npcx9m6f_evb

`./scripts/twister -s tests/kernel/timer/timer_api/kernel.timer.tickless -p npcx9m6f_evb`

gets:

```
[ 86%] Linking C executable app_smem_unaligned_prebuilt.elf
/opt/zephyr-sdk-0.13.0-alpha-1/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: libzephyr.a(power.c.obj): in function `npcx_power_enter_system_sleep':
/home/galak/git/zephyr/soc/arm/nuvoton_npcx/common/power.c:135: undefined reference to `npcx_uart_enable_access_interrupt'
/opt/zephyr-sdk-0.13.0-alpha-1/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: drivers/serial/libdrivers__serial.a(uart_npcx.c.obj): in function `uart_npcx_rx_wk_isr':
/home/galak/git/zephyr/drivers/serial/uart_npcx.c:329: undefined reference to `npcx_uart_disable_access_interrupt'
collect2: error: ld returned 1 exit status
gmake[2]: *** [zephyr/CMakeFiles/app_smem_unaligned_prebuilt.dir/build.make:125: zephyr/app_smem_unaligned_prebuilt.elf] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:3229: zephyr/CMakeFiles/app_smem_unaligned_prebuilt.dir/all] Error 2
```

--- a/drivers/serial/uart_npcx.c
+++ b/drivers/serial/uart_npcx.c
@@ -549,7 +549,6 @@ NPCX_UART_IRQ_CONFIG_FUNC(inst)
 
 DT_INST_FOREACH_STATUS_OKAY(NPCX_UART_INIT)
 
-#ifdef CONFIG_PM_DEVICE
 #define ENABLE_MIWU_CRIN_IRQ(inst)                                             \
 	npcx_miwu_irq_get_and_clear_pending(&uart_npcx_cfg_##inst.uart_rx_wui);\
 	npcx_miwu_irq_enable(&uart_npcx_cfg_##inst.uart_rx_wui);
@@ -566,4 +565,3 @@ void npcx_uart_disable_access_interrupt(void)
 {
 	DT_INST_FOREACH_STATUS_OKAY(DISABLE_MIWU_CRIN_IRQ)
 }
-#endif /* CONFIG_PM_DEVICE */

